isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -24,30 +24,132 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
 -->
   <h2>Overview</h2>
 
+  <p>
+    This installation guide is for ISIS3 users interested in installing ISIS3 (3.6.0)+ through conda.  
+    If you are a developer, we refer you to our <a href="https://github.com/USGS-Astrogeology/ISIS3">GitHub repository.</a>
+  </p>
+
+
+  <h3>ISIS3 Installation With Conda</h3>
+    <ol>
+      <li>Download either the Anaconda or Miniconda installation script for your OS platform.  Anaconda is a much larger distribtion of packages supporting scientific python, while Miniconda is a minimal installation and not as large: 
+      <a href="https://www.anaconda.com/download">Anaconda installer</a>, <a href="https://conda.io/miniconda.html">Miniconda installer</a></li>
+      <li>Open a terminal window in the directory where you downloaded the script, and run the following commands.  In this example, we chose to do a full install of Anaconda, and our OS is Linux-based.  Your file name may be different depending on your environment.</li>
+        <pre>
+          chmod +x Anaconda3-5.2.0-Linux-x86_64.sh
+          ./Anaconda3-5.2.0-Linux-x86_64.sh          
+        </pre>
+        This will start the Anaconda installer which will guide you through the installation process.
+      <li>After the installation has finished, open up a bash prompt in your terminal window.</li>
+      <li>Create a new environment for your ISIS3 installation:
+        <pre>
+          #Add the following channels to the Anaconda installation
+          conda config --add channels conda-forge
+          conda config --add channels usgs-astrogeology
```

Comment:
I believe this can be one command

Reply:
conda config --add channels conda-forge --add-channels usgs-astrogeology